apps/frontend/src/app/3_organisms/FundingHistoryFrame/FundingHistoryFrame.tsx

```diff
@@ -0,0 +1,207 @@
+
+  const [orderOptions, setOrderOptions] = useState<OrderOptions>({
+    orderBy: 'updatedAtTimestamp',
+    orderDirection: OrderDirection.Desc,
+  });
+
+  const { data, loading, refetch } = useGetFundingHistory(
+    account,
+    pageSize,
+    page,
+    orderOptions,
+  );
+
+  useEffect(() => {
+    refetch();
+  }, [refetch, block]);
+
+  const [getFundingHistory] = useGetFundingLazyQuery({
+    client: rskClient,
+  });
+
+  const funding = useMemo(
+    () => (data?.bitcoinTransfers as BitcoinTransfer[]) || [],
+    [data],
+  );
+
+  useEffect(() => {
+    if (account && funding) {
+      //split each row returned from the graph into 2 rows (part 1 and part 2)
+      const data: FundingHistoryType[] = funding.reduce(
+        (acc: FundingHistoryType[], item) => {
+          const row1 = {
+            timestamp: item.createdAtTimestamp,
+            type: BitcoinTransferDirection.Incoming,
+            sent: item.totalAmountBTC,
+            received: null,
+            serviceFee: null,
+            txHash: item.bitcoinTxHash,
+          };
+
+          const row2 = {
+            timestamp: item.createdAtTimestamp,
+            type: BitcoinTransferDirection.Outgoing,
+            sent: null,
+            received: item.totalAmountBTC,
+            serviceFee: item.feeBTC,
+            txHash: item.createdAtTx.id,
+          };
+
+          acc.push(row1, row2);
+          return acc;
+        },
+        [],
+      );
+      setFundingHistory(data);
+    }
+  }, [funding, account]);
+
+  const onPageChange = useCallback(
+    (value: number) => {
+      if (fundingHistory.length < pageSize && value > page) {
+        return;
+      }
+      setPage(value);
+    },
+    [page, fundingHistory.length],
+  );
+
+  const isNextButtonDisabled = useMemo(
+    () => !loading && fundingHistory?.length < pageSize,
+    [loading, fundingHistory],
+  );
+
+  const exportData = useCallback(async () => {
+    const { data } = await getFundingHistory({
+      variables: {
+        user: account,
+        skip: 0,
+        pageSize: EXPORT_RECORD_LIMIT,
+        orderBy: orderOptions.orderBy as BitcoinTransfer_OrderBy,
+        orderDirection: orderOptions.orderDirection,
+      },
+    });
+
+    let funding = data?.bitcoinTransfers || [];
+
+    if (!funding || !funding.length) {
+      addNotification({
+        type: NotificationType.warning,
+        title: t(translations.fundingHistory.actions.noDataToExport),
+        content: '',
+        dismissible: true,
+        id: nanoid(),
+      });
+    }
+    //split each row returned from the graph into 2 rows (part 1 and part 2)
+    const fundingData: FundingHistoryType[] = funding.reduce(
+      (acc: FundingHistoryType[], item) => {
+        const row1 = {
+          timestamp: item.createdAtTimestamp,
+          type: t(translations.fundingHistory.transactionType.part1),
+          sent: item.totalAmountBTC,
+          received: null,
+          serviceFee: null,
```

Comment:
Should pass hyphen here instead of null so it matches data visible in table (same thing for line 157)